### PR TITLE
55 checkpoints challenge

### PR DESCRIPTION
**Pull request by TROCADER**
Geofences are placed evenly based on distance. Easy to change both number of geofences (checkpoints) and minimum distance between them. If the distance is not good enough it will check against the next upcoming point instead.

### CODE REVIEW

**Comment by TROCADER** [comment]
Based on #54 . Merging this should merge the other (I think, no promises).

File: client/lib/pages/generated_map_page.dart [hunk 1]
```diff
@@ -248,19 +247,22 @@ Future<void> _getPolyline(LatLng start) async {
     }
   }
 
-  if (!inIntervall) {
-    totalDistance = 'Failed';
-    points = [];
-    reset();
-  }
+  // if (!inIntervall) {
```
Comment (tubj5452) [inline client/lib/pages/generated_map_page.dart]: tillbaka?

File: client/lib/pages/generated_map_page.dart [hunk 2]
```diff
@@ -248,19 +247,22 @@ Future<void> _getPolyline(LatLng start) async {
...
+  //   totalDistance = 'Failed';
+  //   points = [];
+  //   reset();
+  // }
 
+  // Circle
```
Comment (tubj5452) [inline client/lib/pages/generated_map_page.dart:256]: Är det fortf. en circle?

**Review by tubj5452 — CHANGES_REQUESTED** [review]
merge conflicts

File: client/lib/pages/generated_map_page.dart (hunk 1 above)
Comment (TROCADER) [reply client/lib/pages/generated_map_page.dart]: Fixat

Comment (TROCADER) [reply client/lib/pages/generated_map_page.dart]: Återställde till din version av filen

File: client/lib/pages/generated_map_page.dart (hunk 2 above)
Comment (TROCADER) [reply client/lib/pages/generated_map_page.dart:395]: Återställde till din version av filen

**Pull request merged**